modules/microsite/src/main/scala/DocsHelpers.scala

```diff
@@ -74,6 +74,68 @@ object DocsHelpers {
             )
           case _ => ???
         }
+      case (Http4s, GeneratingAServer) =>
+        val (_, codegenDefinitions) = Target.unsafeExtract(
+          Common.prepareDefinitions[ScalaLanguage, Target](CodegenTarget.Server, Context.empty, openAPI, List("definitions"), NonEmptyList.one("support"))
+        )
+        val server = codegenDefinitions.servers.head
+        val q"""
+          class ${oname}[..${tparms}](...${resourceParams}) extends ..${xtends} {
+            ..${stats}
+          }
+        """ = server.serverDefinitions.head
+
+        val routeStats = stats.collectFirst({ case q"""def routes(...$parms): $rtpe = HttpRoutes.of(${Term.Block(List(Term.PartialFunction(cases)))})""" => q"""
+          def routes(...${parms}): ${rtpe} = ${ Term.Apply(
+            fun = q"HttpRoutes.of",
+            args = List(Term.Block(stats = List(Term.PartialFunction(cases = cases.take(2)))))
+          ) }
+        """}).toList
+        List(
+          Some(server.handlerDefinition.toString),
+          Some(""),
+          Some(q"""
+            class ${oname}[..${tparms}](...${resourceParams}) extends ..${xtends} {
+              ..${routeStats};
+              `...`
+            }
+          """.toString)
+        )
+      case (Http4s, GeneratingClients) if false =>
```

Comment:
lol, explicitly disabled this to test `Unknown docs sample` but then forgot to uncomment it. Fixing in #1151